install-cron-service.sh

```diff
@@ -1,85 +1,137 @@
 #!/bin/bash
 
 function usage() {
-    echo "### Script for install new service for periodically run ###"
+    echo "### Script to install new service to run on schedule ###"
     echo "Usage: $(basename "$0") <website> <artifact> [--suffix <suffix>] <version> [--aes-password <aes password>] <schedule> <jar launch args>"
```

Comment:
I think we should remove this line and leave only usage examples for syncs3 and monitoring-refresh-config

Reply:
Done.